apps/settings/elements/about.html

```diff
@@ -58,30 +58,12 @@ <h2 data-l10n-id="softwareUpdates"></h2>
       </ul>
 
       <header>
-        <h2 data-l10n-id="aboutBrowserOS"></h2>
+        <h2 data-l10n-id="aboutFirefoxOSTitle"></h2>
```

Comment:
Don't change this, please. "FirefoxOS" is a trademark and is not open source. Anyone can use this code, but only those with licensing agreements with Mozilal can use "Firefox" and "FirefoxOS". That's why you sometimes see things like "aboutBrowserOS" in our code.